src/api/routes/meetingSlot.ts

```diff
@@ -0,0 +1,29 @@
+import { procedure, router } from "../trpc";
+import { z } from "zod";
+import db from "../database/db";
+
+export const meetingSlotRouter = router({
+  list: procedure.query(async () => {
+    return await db.MeetingSlot.findAll({
```

Comment:
need permission checking. please refer to other routes that requires admin privliege